kong/pdk/init.lua

```diff
@@ -147,7 +143,7 @@
 -- Instance of Kong's DNS resolver, a client object from the
 -- [lua-resty-dns-client](https://github.com/kong/lua-resty-dns-client) module.
 --
--- **Note:** usage of this module is currently reserved to the core or to
+-- **Note:** Usage of this module is currently reserved to the core or to
 -- advanced users.
```

Comment:
Is this still true? (asking for all such notes in this file)

Reply:
:+1: I'd say it's still true.